src/cf.data.pre

```diff
@@ -2304,8 +2304,10 @@ DOC_START
 			Squid, and treat them as unencrypted HTTP messages,
 			becoming the man-in-the-middle.
 
-			The ssl_bump option is required to fully enable
+			The ssl_bump configuration option is required to fully enable
 			bumping of CONNECT requests.
+			Enabling this option requires also specifying either
+			tproxy or intercept
```

Comment:
This new statement is incorrect: http_port supports SslBump in forward proxy mode as well. That mode is the default.

```suggestion
```